src/main/resources/default/assets/scripts/vfs-dropzone-and-modals.js.pasta

```diff
@@ -3,7 +3,7 @@
  * by scireum in Remshalden, Germany
  *
  * Copyright by scireum GmbH
- * http://www.scireum.de - info@scireum.de
+ * http://www.scireum.de - info@@scireum.de
```

Comment:
**Question:** Don't we omit the copyright notice in pasta files normally? Especially in pasta files that are just a part of a bigger file?